### PR TITLE
Delete leftover files after the migration

### PR DESCRIPTION
**Pull request by lyleschemmerling**
the old legacy jekyll stuff no longer sparks joy

### CODE REVIEW

File: package-lock.json
Comment (mooreds) [inline package-lock.json:1]: Are you sure we want to remove this? I thought we needed it for npm ci which is run during the build process.

File: site/_diagrams/_config.plantuml
Comment (mooreds) [inline site/_diagrams/_config.plantuml:1]: farewell plantuml!

**Review by mooreds — CHANGES_REQUESTED** [review]
I think we need package-lock.json? What am I missing?

File: package-lock.json
Comment (lyleschemmerling) [reply package-lock.json:1]: It looks like this file was added by accident. The package.json and the correlated package-lock.json that we care about are in the astro directory, which is the working directory for the `npm ci` command

**Review by mooreds — APPROVED** [review]

**Pull request merged**